### PR TITLE
Updated themeMode docs

### PR DESCRIPTION
**Pull request by AnshumanPadiya**
This PR updates the `themeMode` docs according to the changes made for issue #6397. 

Screenshot of the change - 
themeModeDocUpdate

### CODE REVIEW

File: src/screens/Grommet.js [hunk 1]
```diff
@@ -187,11 +187,12 @@ const GrommetPage = () => (
         <Property name="themeMode">
```
Comment (jcfilben) [inline src/screens/Grommet.js:187]:
```suggestion
        <Property stable name="themeMode">
```
Can we add the stable prop to this? This is something new we are doing. The stable prop just lets people know that this is a new prop that hasn't been released yet. Once we release a new version of grommet we go through and remove all the stable flags from the docs.

File: src/screens/Grommet.js [hunk 2]
```diff
@@ -187,11 +187,12 @@ const GrommetPage = () => (
...
           <Description>
             Dark vs. light theme variation. Default is unspecified and left to
-            theme.
+            theme. Auto sets system preferred theme.
```
Comment (jcfilben) [inline src/screens/Grommet.js:190]:
```suggestion
            theme. Auto defaults to the system preferred theme mode.
```

File: src/screens/Grommet.js (hunk 1 above)
Comment (AnshumanPadiya) [reply src/screens/Grommet.js:187]: Sure thing! I will take that in mind for any future updates.

**Review by jcfilben — APPROVED** [review]
Looks good!

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**